packages/protocol/test/identity/federatedattestations.ts

```diff
@@ -1232,64 +895,68 @@ contract('FederatedAttestations', (accounts: string[]) => {
 
     it('should succeed when >1 identifiers are registered for (account, issuer)', async () => {
       await signAndRegisterAttestation(identifier2, issuer1, account1, nowUnixTime, signer1)
-      await federatedAttestations.deleteAttestation(identifier2, issuer1, account1, {
+      await federatedAttestations.revokeAttestation(identifier2, issuer1, account1, {
         from: account1,
       })
       await assertAttestationNotInStorage(identifier2, issuer1, account1, 0, 1)
       await assertAttestationInStorage(identifier1, issuer1, 0, account1, nowUnixTime, signer1, 0)
     })
 
-    it('should revert if an invalid user attempts to delete the attestation', async () => {
+    const newAttestation = [
+      [0, 'identifier', identifier2],
+      // skipping issuer as it requires a different signer as well
+      [2, 'account', accounts[3]],
+      [3, 'issuedOn', nowUnixTime + 1],
+      [4, 'signer', accounts[3]],
+    ]
+    newAttestation.forEach(([index, arg, newVal]) => {
+      it(`after revoking an attestation, should succeed in registering new attestation with different ${arg}`, async () => {
+        await federatedAttestations.revokeAttestation(identifier1, issuer1, account1)
+        const args = [identifier1, issuer1, account1, nowUnixTime, signer1]
+        args[index] = newVal
+        assert.isOk(signAndRegisterAttestation.apply(this, args))
+      })
+    })
+
+    it('should revert if an invalid user attempts to revoke the attestation', async () => {
       await assertRevert(
-        federatedAttestations.deleteAttestation(identifier1, issuer1, account1, {
+        federatedAttestations.revokeAttestation(identifier1, issuer1, account1, {
           from: accounts[4],
         })
       )
     })
 
-    it('should revert if a revoked signer attempts to delete the attestation', async () => {
-      await federatedAttestations.revokeSigner(signer1)
+    it('should fail registering same attestation and fail again after revoking it', async () => {
```

Comment:
I think this test can be simplified to just `fail to register a revoked attestation` (and placed either with revoke or register tests)? 
`should revert if an attestation with the same (issuer, identifier, account) is uploaded again` already exists within the register tests, and it's probably easier to debug if just the "register, revoke, register" flow is tested here (instead of "register x2, revoke, register"), i.e. delete L930-41